dirty_cat/similarity_encoder.py

```diff
@@ -3,7 +3,7 @@
 import numpy as np
 from scipy import sparse
 from sklearn.feature_extraction.text import CountVectorizer, HashingVectorizer
-from sklearn.preprocessing._encoders import _BaseEncoder
+from sklearn.preprocessing import OneHotEncoder
```

Comment:
@dsleo can I fix this the merge conflict and commit it to your forks master? Like that we can merge this and close the PR :)

Reply:
@mcuny weird, I did that change as suggested by Gael yesterday but sure go ahead.